code/.editorconfig

```diff
@@ -0,0 +1,320 @@
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+indent_style = space
+insert_final_newline = false
+max_line_length = 120
+tab_width = 4
+ij_continuation_indent_size = 4
+ij_formatter_off_tag = @formatter:off
+ij_formatter_on_tag = @formatter:on
+ij_formatter_tags_enabled = false
+ij_smart_tabs = false
+ij_visual_guides = 120
+ij_wrap_on_typing = false
+
+[*.java]
```

Comment:
I've only included the configurations for Java and Kotlin as those were the only two languages I've configured. I manually removed the configs for other languages when exporting the file from Android Studio.

Reply:
Any updates on this one?